docs/src/main/sphinx/admin/fault-tolerant-execution.rst

```diff
@@ -365,7 +365,8 @@ for your storage solution.
      -
    * - ``exchange.s3.endpoint``
      - S3 storage endpoint server if using an S3-compatible storage system that
-       is not AWS. If using AWS S3, can be ignored.
+       is not AWS. If using AWS S3, can be ignored. If using GCS, set it to
```

Comment:
Should we maybe add `Notes` column where we would put stuff like `S3 only`.
Or even more explicitly `Filesystem` where we would list applicable ones or just put `ALL` for general parameters?

Reply:
@colebow : thoughts? Would this be too complicated?

Reply:
I think it'd be pretty reasonable to add another column, especially because several other rows on this table are also restricted. I don't like the idea of specifying 'all' for everything that's universal - I'd rather go with something like "limited to" so you specify what it can be used on, but blank means everything.